### ISSUE TITLE
[QUERY] Microsoft.Azure.WebJobs EventHubTrigger PartitionContext empty except PartitionId

### ISSUE THREAD

**Issue by christophwille** [Azure/azure-sdk-for-net#29810]
### Library name and version

Microsoft.Azure.WebJobs.Extensions.EventHubs 5.1.1

### Query/Question

Porting from 4.0.1

Previously, PartitionContext had even methods, but now it is properties only.  Method sig is simplified:

`EventData[] events, PartitionContext PartitionContext`

What I get though is this:
![image](https://user-images.githubusercontent.com/344208/178477272-746427a0-fcb1-4c03-ae3d-2dc5968d91ad.png)

Basically everything empty except for PartitionId - is this supposed to be the case? Esp. EventHubName and ConsumerGroup is very surprising to be empty (and we rely on that in our code). One internal var seems to have most information... any help appreciated.

**Comment by christophwille** [comment]
I started digging a bit (namespace Azure.Messaging.EventHubs.Consumer):

```
        /// <remarks>
        ///   This overload should no longer be used; it does not set the members of
        ///   the context not specified.
        /// </remarks>
        [EditorBrowsable(EditorBrowsableState.Never)]
        protected PartitionContext(string partitionId)
        {
            Argument.AssertNotNullOrEmpty(partitionId, nameof(partitionId));
            PartitionId = partitionId;
        }
```

But (namespace Microsoft.Azure.WebJobs.EventHubs.Processor)

```
        private class EventProcessorHostPartitionContext : PartitionContext
        {
            private readonly EventProcessorHostPartition _hostPartition;

            public EventProcessorHostPartitionContext(EventProcessorHostPartition hostPartition) : base(hostPartition.PartitionId)
            {
                _hostPartition = hostPartition;
            }

            public override LastEnqueuedEventProperties ReadLastEnqueuedEventProperties() => _hostPartition.LastEnqueuedEventProperties;
        }
```

Do I take that the remarks section never made it to WebJobs' use of the base class ctors?

**Comment by jsquire** [comment]
Hi @christophwille.   Thank you for reaching out and we regret that you're experiencing difficulties.   You're correct in that the extensions package was overlooked and isn't passing the additional context parameters.   There's a fix out and this will be available in the next release.
